final_task/rss_reader/database.py

```diff
@@ -0,0 +1,121 @@
+import datetime
+from contextlib import closing
+
+import psycopg2
+import News
+import os
+import logging
+
+MODULE_LOGGER = logging.getLogger("rss_reader.database")
+
+
+def get_param_for_connect(filename) -> dict:
+    logger = logging.getLogger("rss_reader.database.get_param_for_connect")
+    logger.info("get param for connect from config.txt")
+    dict_parameters = {}
+    with open(filename, "r") as file:
+        for line in file:
+            key, value = line.split()
+            dict_parameters[key] = value
+    return dict_parameters
+
+
+def connect_to_database():
+    logger = logging.getLogger("rss_reader.database.connect_to_database")
+    logger.info("connect to database")
+    if os.path.isfile('final_task/config.txt'):
+        filename = "final_task/config.txt"
+    elif os.path.isfile('config.txt'):
+        filename = 'config.txt'
+    else:
+        raise psycopg2.OperationalError("check config")
+
+    parameters = get_param_for_connect(filename)
+    con = psycopg2.connect(
+        database=parameters['database'],
+        user=parameters['user'],
+        password=parameters['password'],
+        host=parameters['host'],
+        port=parameters['port']
+    )
+    return con
+
+
+def is_table():
+    logger = logging.getLogger("rss_reader.database.is_table")
+    logger.info("check exist table")
+    flag_is_table = True
+    with closing(connect_to_database()) as con:
+        cursor = con.cursor()
+
+        try:
+            cursor.execute("SELECT * FROM NEWS")
+        except psycopg2.DatabaseError:
+            flag_is_table = False
+        finally:
+            con.close()
```

Comment:
A question:

Do you really need this `finally` block if you using context manager?